### PR TITLE
fix(apiv2): update host timeline listing

### PR DESCRIPTION
**Pull request by victorvassilev**
## Description

Address feedback after validation.

**Fixes** # MON-4867

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: src/Centreon/Domain/Monitoring/Entity/LogEventObject.php [hunk 1]
```diff
@@ -85,22 +101,17 @@ public static function loadEntityDescriptorMetadata(): array
...
-            default:
-                $textValue = null;
-                break;
+        $textValue = null;
+        if ($this->getServiceId() === 0) {
+            //calculate status for host
+            if (isset(self::STATUS_LIST_HOST[$this->getStatus()])) {
+                $textValue = self::STATUS_LIST_HOST[$this->getStatus()];
+            }
+        } else {
```
Comment (vhr) [inline src/Centreon/Domain/Monitoring/Entity/LogEventObject.php]:
```suggestion
        } else if (isset(self::STATUS_LIST_SERVICE[$this->getStatus()])) {
```

File: src/Centreon/Domain/Monitoring/Entity/LogEventObject.php [hunk 2]
```diff
@@ -85,22 +101,17 @@ public static function loadEntityDescriptorMetadata(): array
...
+            //calculate status for service
+            if (isset(self::STATUS_LIST_SERVICE[$this->getStatus()])) {
```
Comment (vhr) [inline src/Centreon/Domain/Monitoring/Entity/LogEventObject.php]:
```suggestion
```

File: src/Centreon/Domain/Monitoring/Entity/LogEventObject.php [hunk 3]
```diff
@@ -85,22 +101,17 @@ public static function loadEntityDescriptorMetadata(): array
...
-                $textValue = null;
-                break;
+        $textValue = null;
+        if ($this->getServiceId() === 0) {
+            //calculate status for host
+            if (isset(self::STATUS_LIST_HOST[$this->getStatus()])) {
+                $textValue = self::STATUS_LIST_HOST[$this->getStatus()];
+            }
+        } else {
+            //calculate status for service
```
Comment (vhr) [inline src/Centreon/Domain/Monitoring/Entity/LogEventObject.php]:
```suggestion
        //calculate status for service
```

File: src/Centreon/Domain/Monitoring/Entity/LogEventObject.php [hunk 4]
```diff
@@ -85,22 +101,17 @@ public static function loadEntityDescriptorMetadata(): array
...
+            if (isset(self::STATUS_LIST_SERVICE[$this->getStatus()])) {
+                $textValue = self::STATUS_LIST_SERVICE[$this->getStatus()];
```
Comment (vhr) [inline src/Centreon/Domain/Monitoring/Entity/LogEventObject.php]:
```suggestion
        $textValue = self::STATUS_LIST_SERVICE[$this->getStatus()];
```

File: src/Centreon/Domain/Monitoring/Entity/LogEventObject.php [hunk 5]
```diff
@@ -85,22 +101,17 @@ public static function loadEntityDescriptorMetadata(): array
...
+            }
```
Comment (vhr) [inline src/Centreon/Domain/Monitoring/Entity/LogEventObject.php]:
```suggestion
```

File: src/Centreon/Domain/Monitoring/Entity/LogEventObject.php [hunk 6]
```diff
@@ -85,22 +101,15 @@ public static function loadEntityDescriptorMetadata(): array
...
-                break;
+        $textValue = null;
+        if ($this->getServiceId() === 0) {
+            //calculate status for host
+            if (isset(self::STATUS_LIST_HOST[$this->getStatus()])) {
+                $textValue = self::STATUS_LIST_HOST[$this->getStatus()];
+            }
+        } else if (isset(self::STATUS_LIST_SERVICE[$this->getStatus()])) {
+        //calculate status for service
+        $textValue = self::STATUS_LIST_SERVICE[$this->getStatus()];
```
Comment (kduret) [inline src/Centreon/Domain/Monitoring/Entity/LogEventObject.php]: bad indentation

Comment (victorvassilev) [reply src/Centreon/Domain/Monitoring/Entity/LogEventObject.php]: thanks, fixed

**Pull request merged**